text/0000-opt-in-layer-caching.md

```diff
@@ -0,0 +1,53 @@
+# Meta
+[meta]: #meta
+- Name: Opt-in Layer Caching
+- Start Date: 2020-07-26
+- Author(s): @sclevine
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Restored layers are currently re-cached automatically. This RFC proposes that buildpacks must opt-in to re-caching.
+
+# Motivation
+[motivation]: #motivation
+
+- If the latest version of the buildpack is not aware of layers created by **any** earlier version of the buildpack and it doesn't removed all unrecognized layers, those layers may stick around indefinitely. This may lead to unexpected behavior and/or security vulnerabilities.
```

Comment:
```suggestion
- If the latest version of the buildpack is not aware of layers created by **any** earlier version of the buildpack and it doesn't remove all unrecognized layers, those layers may stick around indefinitely. This may lead to unexpected behavior and/or security vulnerabilities.
```